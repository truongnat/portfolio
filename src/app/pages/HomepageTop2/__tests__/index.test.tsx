import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { HomePageTop2 } from '..';

const shallowRenderer = createRenderer();

describe('<HomePageTop2 />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<HomePageTop2 />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
