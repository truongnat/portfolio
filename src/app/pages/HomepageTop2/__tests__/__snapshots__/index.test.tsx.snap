// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePageTop2 /> should render and match the snapshot 1`] = `
	<React.Fragment>
      <Helmet>
        <title>Home Page</title>
        <meta name="Home Page" content="My page for Truong.Nat" />
      </Helmet>
      <Wrapper>
        <WrapParticles
          params={{
            particles: {
              number: {
                value: particlesVal,
              },
              size: {
                value: 1,
              },
            },
            interactivity: {
              events: {
                onhover: {
                  enable: true,
                  mode: 'repulse',
                },
              },
            },
          }}
        />
        <Container>
          <Title>Truong.Nat</Title>
          <SubTitle>WORKER | DEVELOPER | SHIPPER</SubTitle>
          <NavMobile />
          <NavDesktop />
        </Container>
      </Wrapper>
    </React.Fragment>
`